app/models/Organization.ts

```diff
@@ -74,3 +83,74 @@ export const getResourceLocations = (org: Organization): LocationDetails[] => {
     recurringSchedule: org.recurringSchedule,
   }));
 };
+
+// TODO Can we remove this?
```

Comment:
Since we still have the API to verify in the backend, I'd wondered whether removing this should be a standalone issue.

But agreed, removing, can re-add clean if required down the line.